pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pairwise-compare"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
Consider using [`poetry-dynamic-versioning`](https://github.com/mtkennerly/poetry-dynamic-versioning) to instead rely on git tags for versions. If you tie this to GitHub releases you can also use GH Actions to automatically deploy your package to PyPI by way of [trusted publishing](https://docs.pypi.org/trusted-publishers/). The advantage becomes that you aren't tied to making a pull request to modify the version for new releases.